Cargo.toml

```diff
@@ -23,7 +23,6 @@ default-members = [
     "common",
     "nexus",
     "nexus/src/db/db-macros",
-    "nexus/test-utils",
```

Comment:
Note: this has been removed from the *default members*, but we're definitely still using / testing it.

This just means that `cargo build` doesn't, by default, build a "test-only" helper crate - doing so would also require "cockroachdb" to be on the PATH earlier for some of our checks.

Note: This still gets built with `cargo build --all-targets` and `cargo test`, so no worries on coverage.